MdsSupportingClasses/MdsColliveryService.php

```diff
@@ -584,9 +584,17 @@ public function automatedAddCollivery($order_id, $processing = false)
                     $collivery_to = $address['address_id'];
                     $contact_to = $address['contact_id'];
 
-                    $instructions = 'Order number: ' . $order_id;
+                    $instructions = '';
+                    if ($this->settings->getValue('include_customer_note') == 'yes') {
+                        $instructions .= $order->get_customer_note();
+                    }
+                    if ($this->settings->getValue('include_order_number') == 'yes') {
+                        if(strlen($instructions)>0)$instructions .= ' ';
+                        $instructions .= 'Order number: ' . $order_id;
+                    }
                     if ($this->settings->getValue('include_product_titles') == 'yes') {
                         $count = 1;
+                        if(strlen($instructions)>0)$instructions .= ' ';
```

Comment:
Please modify this syntax into:
```
if () {
  $instrunctions .= ' ';
}
```

Reply:
93c631c9188d5a558d1246baf7ea93d007562317